frontend/src/pages/Welcome/index.js

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useTheme } from "styled-components";
+import {
+  StyledWelcome,
+  StyledWelcomeWrapper,
+  StyledWelcomeImage,
+  StyledWelcomeHeader,
+  StyledWelcomeButton,
+} from "./styled";
+import congrats from '../../images/congrats.jpg';
+
+const Welcome = () => {
+  const theme = useTheme();
```

Comment:
you don't need to utilize useTheme here
I know you are using it for the button bellow .. but since passed the possible colors in the theme provider, you can basically expect them in the body of the string of the styled.component.. in styled.js
a further comment will show you how